docs/architecture/psa-crypto-repository.md

```diff
@@ -0,0 +1,17 @@
+PSA Cryptography repository
+===========================
+## Introduction
+
+The PSA Cryptography repository contains a reference implementation of the [PSA Cryptography API specification](https://armmbed.github.io/mbed-crypto/psa/#application-programming-interface).
+
+## Requirements
+
+* The PSA Cryptography repository exposes as public interface the cryptographic interface defined in the PSA Cryptography API specification and solely this interface.
+* The PSA Cryptography repository provides a way to build and test a C static and/or shared library exposing completely or partially the PSA Cryptography API.
+* The PSA Cryptography repository provides a way to define the parts of the PSA Cryptography API exposed by the built C library.
```

Comment:
What does "provides a way to define..." mean?

Reply:
I am changing to "provides a configuration mechanism to define ...".